packages/@office-iss/react-native-win32/src/Libraries/Types/CoreEventTypes.win32.js

```diff
@@ -83,28 +83,144 @@ export type TextLayoutEvent = SyntheticEvent<
   |}>,
 >;
 
-export type PointerEvent = ResponderSyntheticEvent<
-  $ReadOnly<{|
-    pointerId: number,
-    pressure: number,
-    pointerType: string,
-    clientX: number,
-    clientY: number,
-    target: ?number,
-    timestamp: number,
-    // [Windows
-    isLeftButton: boolean,
-    isRightButton: boolean,
-    isMiddleButton: boolean,
-    isBarrelButtonPressed: boolean,
-    isHorizontalScrollWheel: boolean,
-    isEraser: boolean,
-    shiftKey: boolean,
-    ctrlKey: boolean,
-    altKey: boolean,
-    // Windows]
-  |}>,
->;
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/API/UIEvent
+ */
+export interface NativeUIEvent {
+  /**
+   * Returns a long with details about the event, depending on the event type.
+   */
+  +detail: number;
+}
+
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
+ */
+export interface NativeMouseEvent extends NativeUIEvent {
+  /**
+   * The X coordinate of the mouse pointer in global (screen) coordinates.
+   */
+  +screenX: number;
+  /**
+   * The Y coordinate of the mouse pointer in global (screen) coordinates.
+   */
+  +screenY: number;
+  /**
+   * The X coordinate of the mouse pointer relative to the whole document.
+   */
+  +pageX: number;
+  /**
+   * The Y coordinate of the mouse pointer relative to the whole document.
+   */
+  +pageY: number;
+  /**
+   * The X coordinate of the mouse pointer in local (DOM content) coordinates.
+   */
+  +clientX: number;
+  /**
+   * The Y coordinate of the mouse pointer in local (DOM content) coordinates.
+   */
+  +clientY: number;
+  /**
+   * Alias for NativeMouseEvent.clientX
+   */
+  +x: number;
+  /**
+   * Alias for NativeMouseEvent.clientY
+   */
+  +y: number;
+  /**
+   * Returns true if the control key was down when the mouse event was fired.
+   */
+  +ctrlKey: boolean;
+  /**
+   * Returns true if the shift key was down when the mouse event was fired.
+   */
+  +shiftKey: boolean;
+  /**
+   * Returns true if the alt key was down when the mouse event was fired.
+   */
+  +altKey: boolean;
+  /**
+   * Returns true if the meta key was down when the mouse event was fired.
+   */
+  +metaKey: boolean;
+  /**
+   * The button number that was pressed (if applicable) when the mouse event was fired.
+   */
+  +button: number;
+  /**
+   * The buttons being depressed (if any) when the mouse event was fired.
+   */
+  +buttons: number;
+  /**
+   * The secondary target for the event, if there is one.
+   */
+  +relatedTarget: null | number | React.ElementRef<HostComponent<mixed>>;
+
+  // [Windows
+  +isLeftButton: boolean;
+  +isRightButton: boolean;
+  +isMiddleButton: boolean;
+  +isBarrelButtonPressed: boolean;
+  +isHorizontalScrollWheel: boolean;
+  +isEraser: boolean;
+  +target: ?number;
+  // Windows]
+}
```

Comment:
[PR: Fill out w3c pointer events](https://github.com/facebook/react-native/commit/291f26c5c0d8ee4785bbd0bc344e0341115fe63f). Specifically wanted to  call out this PR, it changed around some properties for PointerEvent, splitting it into NativeUIEvent, NativeMouseEvent, and NativePointerEvent.

We already had windows specific events in PointerEvent, so I added those (and the target property which they removed) to NativeMouseEvent. Let me know if this is how we want to handle this or if there's a better way!